client/app/dedicatedCloud/security/dedicatedCloud-security.html

```diff
@@ -153,6 +153,17 @@ <h2 data-translate="dedicatedCloud_tab_security"></h2>
                             <span data-translate="{{:: 'dedicatedCloud_vm_encryption_kms_status_' + $row.state }}"></span>
                         </span>
                     </oui-column>
+                    <oui-action-menu aria-label="{{:: 'common_actions' | translate }}"
+                                     data-compact>
+                        <oui-action-menu-item aria-label="{{:: 'dedicatedCloud_vm_encryption_edit_kms' | translate }}"
```

Comment:
Why do you use an `aria-label` ? The content is already described in the span below